tests/integ/test_pandas_to_df.py

```diff
@@ -336,14 +354,11 @@ def test_write_pandas_with_timestamps(session):
         ],
         columns=["tm_tz", "tm_ntz"],
     )
-    table_name = Utils.random_name_for_temp_object(TempObjectType.TABLE)
```

Comment:
Let's use `local_testing_mode` in an if-else clause and keep the deleted code just so we could test `write_pandas`.

Reply:
updated